client/i18n/schema-validation.ts

```diff
@@ -1,27 +1,35 @@
-const path = require('path');
-const { availableLangs } = require('../../config/i18n/all-langs');
-const introSchema = require('./locales/english/intro.json');
-const linksSchema = require('./locales/english/links.json');
-const metaTagsSchema = require('./locales/english/meta-tags.json');
-const motivationSchema = require('./locales/english/motivation.json');
-const translationsSchema = require('./locales/english/translations.json');
-const trendingSchema = require('./locales/english/trending.json');
+import path from 'path';
+import { availableLangs } from '../../config/i18n/all-langs';
+import introSchema from './locales/english/intro.json';
+import linksSchema from './locales/english/links.json';
+import metaTagsSchema from './locales/english/meta-tags.json';
+import motivationSchema from './locales/english/motivation.json';
+import translationsSchema from './locales/english/translations.json';
+import trendingSchema from './locales/english/trending.json';
+
+type MotivationalQuotes = { quote: string; author: string }[];
 
 /**
  * Flattens a nested object structure into a single
  * object with property chains as keys.
  * @param {Object} obj Object to flatten
  * @param {String} namespace Used for property chaining
  */
-const flattenAnObject = (obj, namespace = '') => {
-  const flattened = {};
+const flattenAnObject = (
+  obj: Record<string, object | string>,
```

Comment:
To keep things consistent. Also, If we use `object`, we might as well make the full type `object`. However, we avoid using `object`.

```suggestion
  obj: Record<string, unknown>,
```

Reply:
have modified it to `obj: Record<string, unknown>`